Assets/Oculus/Voice/Scripts/Runtime/Utilities/MicPermissionsManager.cs

```diff
@@ -35,12 +36,16 @@ public static bool HasMicPermission()
     #endif
         }
     
-        public static void RequestMicPermission()
+        public static void RequestMicPermission(Action<string> permissionGrantedCallback = null)
```

Comment:
This has a high probability of getting clobbered by voice sdk updates. We should make this change in the app code until this can be merged into wit-unity or vsdk.

Reply:
I'm pushing the same changes to the SDK as well.